docs/install_guides/installing-openvino-from-archive-linux.md

```diff
@@ -10,23 +10,54 @@ See the [Release Notes](https://software.intel.com/en-us/articles/OpenVINO-RelNo
 
 
 @sphinxdirective
+
 .. tab:: System Requirements
 
-   | Full requirement listing is available in:
-   | `System Requirements Page <https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/system-requirements.html>`_
+  | Full requirement listing is available in:
+  | `System Requirements Page <https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/system-requirements.html>`_
 
 .. tab:: Processor Notes
 
   Processor graphics are not included in all processors. 
   See `Product Specifications`_ for information about your processor.
-  
+
   .. _Product Specifications: https://ark.intel.com/
 
 .. tab:: Software
 
   * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
-  * GCC 7.5.0 (for Ubuntu 18.04) or GCC 9.3.0 (for Ubuntu 20.04)
   * `Python 3.7 - 3.10, 64-bit <https://www.python.org/downloads/>`_
+  * GCC
```

Comment:
<img width="983" alt="Screenshot 2023-02-22 at 14 23 41" src="https://user-images.githubusercontent.com/4737252/220592740-14c44c2f-4a10-4a91-9472-eac048f8160d.png">
Can we add indentation for the second tab group?
```suggestion
  * GCC:
```